### PR TITLE
MemcacheAdapter: using 1 as default weight instead of 10

### PR DESCRIPTION
**Pull request by bryanlatten**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,5 +9,4 @@
 phpunit.xml
 
 # Vendor Assets
-composer.lock
```
Comment (bryanlatten) [inline .gitignore]: Adding this file back in. Rate limits get hit too often without it

File: src/Adapters/MemcacheAdapter.php [hunk 2]
```diff
@@ -12,7 +12,7 @@ class MemcacheAdapter extends AdapterAbstract {
    * @see http://php.net/manual/en/memcache.addserver.php
    */
   const DEFAULT_PERSISTENT           = true;
-  const DEFAULT_WEIGHT               = 10;
+  const DEFAULT_WEIGHT               = 1;
```
Comment (bryanlatten) [inline src/Adapters/MemcacheAdapter.php]: Causes backend to strangely not see keys, somehow.

**Comment by jimdelois** [comment]
:white_check_mark:

**Comment by bryanlatten** [comment]
@be-dmitry this one's for you.

**Pull request merged**